pyiron_workflow/nodes/composite.py

```diff
@@ -27,6 +27,29 @@
     from pyiron_workflow.storage import StorageInterface
 
 
+def get_graph_as_dict(composite: Composite) -> dict:
+    if not hasattr(composite, "_children"):
```

Comment:
I would be more comfortable with a direct class check here rather than the implicit check on a private field encoded as a string